libbeat/outputs/kafka/config.go

```diff
@@ -45,10 +46,10 @@ type metaRetryConfig struct {
 
 var (
 	defaultConfig = kafkaConfig{
-		Hosts:   nil,
-		TLS:     nil,
-		Timeout: 30 * time.Second,
-		Worker:  1,
+		Hosts:       nil,
+		TLS:         nil,
+		Timeout:     30 * time.Second,
+		BulkMaxSize: 2048,
```

Comment:
worker feature?

The `worker` setting was (and still is) indirectly implemented by ReadHostList. If worker > 0, the function just duplicates the hostnames. But for the kafka output it doesn't really makes sense to use worker.